src/compiler/core.ts

```diff
@@ -2987,18 +2987,19 @@ namespace ts {
     }
 
     /** Remove the *first* occurrence of `item` from the array. */
-    export function unorderedRemoveItem<T>(array: T[], item: T): void {
+    export function unorderedRemoveItem<T>(array: T[], item: T) {
         unorderedRemoveFirstItemWhere(array, element => element === item);

```

Comment:
this still does not return a value.. did you mean to change this to `return unorderedRemoveFirstItemWhere(array, element => element === item);`